includes/class-assets.php

```diff
@@ -208,6 +210,24 @@ public function load_css() {
         );
     }
 
+	/**
+	 * Image listing css styling.
+	 * This sets the dimensions of the listing image if not cropped.
+	 *
+	 * @since x.x
+	 */
+	public function image_listing_css() {
+		$thumbnail_width      = wpbdp_get_option( 'thumbnail-width' );
+		$thumbnail_height     = wpbdp_get_option( 'thumbnail-height' );
+		wp_add_inline_style(
+			'wpbdp-base-css',
+			'.wpbdp-listing .listing-thumbnail img {
+				max-width: ' . $thumbnail_width .'px;
+				max-height: ' . $thumbnail_height .'px;
```

Comment:
These need escaping. Also, we could use the 'Crop thumbnails to exact dimensions?' setting. If that is true, we'll need CSS cropping rather than just max sizes.

I think this would be cleaner if we keep it all together in the load_css() function. We should add vars to that rather than extra inline css.

Reply:
Thanks for the changes :) I have updated the css. Will continue on the tests on the images

Reply:
Thanks Paul. There will be more to this than just the height and width. This would be best to add css vars, just like we did in that same function. Then we'll use them in the css. If cropping is happening, we need to use "object-fit: cover" to keep images proportionate. Please use your browser tools to inspect how we are doing this on the submit listing form.

Reply:
Will do :)

Reply:
Did some changes and added this CSS 
```
.wpbdp-thumbnail {
    max-width: var(--bd-thumbnail-width);
    max-height: var(--bd-thumbnail-height);
    &-cropped {
        width: var(--bd-thumbnail-width);
        height: var(--bd-thumbnail-height);
        object-fit: cover;
    }
}
```

Reply:
With cropped enabled, the class `wpbdp-thumbnail-cropped` is added